File: C programming/Assignment/Assignment1/evenodd.c
#include <stdio.h>
void main(){
    int num;
    printf("enter the number:");
    scanf("%d",&num);
    if(num>0){
       if(num%2==0){
           printf("even");
        }
       else{
           printf("odd");
        }
    }
    else{
        printf("enter valid number.");
    }
}

File: Java_programming/basic_java/assignment1/evenodd.java
package assignment1;

public class evenodd {

	public static void main(String[] args) {
		int num=12;
//	    printf("enter the number:");
//	    scanf("%d",&num);
	    if(num>0){
	       if(num%2==0){
	           System.out.printf("even");
	        }
	       else{
	    	   System.out.printf("odd");
	        }
	    }
	    else{
	    	System.out.printf("enter valid number.");
	    }

	}

}
